src/main/java/sayPotato/sound/SoundSaver.java

```diff
@@ -0,0 +1,38 @@
+package sayPotato.sound;
+
+import java.io.*;
+import javax.swing.*;
+import javax.sound.sampled.*;
+import javax.swing.filechooser.FileNameExtensionFilter;
+
+
+public class SoundSaver {
+
+    private JFileChooser chooser = new JFileChooser();
+    private AudioFileFormat.Type fileType = AudioFileFormat.Type.WAVE;
+    private String path;
+    private File wavFile;
+
+    public void saveRecord(byte [] record, AudioFormat format){
```

Comment:
W jednej funkcji mamy zarówno zapis do pliku jak i fragment interfejsu graficznego (file chooser). Powinniśmy to rozdzielić na osobne funkcje. Ta funkcja powinna przyjmować trzeci argument, który byłby obiektem klasy File.